README.md

```diff
@@ -8,75 +8,102 @@ We more than welcome contributions to KubeVirt documentation. Please reach out i
 
 ### Get started
 
-- Create fork of GitHub user-guide repo
+1. Create fork of GitHub user-guide repo
 
-- Clone repository, check out source branch and prepare the Jekyll site
-  ```console
-  git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
-  ```
+2. Clone repository, check out source branch and prepare the mkdocs site
 
-- Set up your git repo remotes like the following:
+```console
+$ git clone -b master https://github.com/mygithubname/user-guide.git && cd user-guide
 ```
+
+3. Set up your git repo remotes like the following:
+
+```console
 $ git remote -v
 origin	git@github.com:mygithubname/user-guide.git (fetch)
 origin	git@github.com:mygithubname/user-guide.git (push)
 upstream	git@github.com:kubevirt/user-guide.git (fetch)
 upstream	git@github.com:kubevirt/user-guide.git (push)
 ```
 
-- We use [mkdocs](https://www.mkdocs.org/) markdown engine with [mkdocs-awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/) plugin
+We use [mkdocs](https://www.mkdocs.org/) markdown engine with [mkdocs-awesome-pages](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin/) plugin
```

Comment:
No numbered bullet point here?

Reply:
Ah this seemed more like a note/regular sentence to me and not a completely new point that had something to do with the other numbered points (i.e. setting up/contributing). It was a bit weird to use a bullet point just floating in space.